docs/admin/logging/log_tags.md

```diff
@@ -46,6 +46,21 @@ For advanced usage, the generated tag's use [go
 templates](http://golang.org/pkg/text/template/) and the container's [logging
 context](https://github.com/docker/docker/blob/master/daemon/logger/context.go).
 
+As an example of what is possible with the syslog logger:
+
+```
+$ docker run -it --rm --log-driver syslog --log-opt tag="{{ (.ExtraAttributes nil).MESOS_TASK_ID }}" --log-opt env=MESOS_TASK_ID -e MESOS_TASK_ID=logtester.1234 flyinprogrammer/logtester
```

Comment:
I built the docs for this, and I think we should wrap this, otherwise only a small part of the example is visible;

<img width="708" alt="screen shot 2016-04-08 at 17 20 07" src="https://cloud.githubusercontent.com/assets/1804568/14400537/54e67af2-fdae-11e5-8842-5cbfe91e8a68.png">

Also, I suggest to change the label name to something that more clearly stands out as being for "illustrational" purposes (e.g. `SOME_ENV_VAR`), so:

```
$ docker run -it --rm \
    --log-driver syslog \
    --log-opt tag="{{ (.ExtraAttributes nil).SOME_ENV_VAR }}" \
    --log-opt env=SOME_ENV_VAR \
    -e SOME_ENV_VAR=logtester.1234 \
    flyinprogrammer/logtester
```

(I like the idea of the log tester image BTW!)